cloud/src/common/config.h

```diff
@@ -145,4 +145,7 @@ CONF_Bool(focus_add_kms_data_key, "false");
 CONF_Bool(enable_txn_store_retry, "false");
 CONF_Int32(txn_store_retry_times, "20");
 
+// The secondary package name of the MetaService.
+CONF_String(secondary_package_name, "selectdb");
```

Comment:
the default should be empty.
And better to explain why need this config and how to use, for other opensource user who want to expand this service